easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.2.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.2.0'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
+
+description = """Xerces-C++ is a validating XML parser written in a portable
+subset of C++. Xerces-C++ makes it easy to give your application the ability to
+read and write XML data. A shared library is provided for parsing, generating,
+manipulating, and validating XML documents using the DOM, SAX, and SAX2
+APIs."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [
+    'http://archive.apache.org/dist/xerces/c/%(version_major)s/sources/'
+]
+sources = ['xerces-c-%(version)s.tar.gz']
+checksums = ['d3162910ada85612f5b8cc89cdab84d0ad9a852a49577691e54bc7e9fc304e15']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.30'),
+]
+
+dependencies = [
+    ('cURL', '7.60.0'),
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
```

Comment:
The OpenSSL OS dependency is already required for `cURL`. Is there a specific reason why you added it here again? I parsed the configure output and it does not seem to look for any library provided by OpenSSL.